test_scripts/API/ButtonSubscription/032_SubscribeButton_Unexpected_disconect_HMI_doesn_t_response_timeout.lua

```diff
@@ -0,0 +1,65 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:
+-- https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0192-button_subscription_response_from_hmi.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) Mobile app is subscribed for button_1
+-- 2) Unexpected disconnect and connect are performed
+-- 3) App registers with actual hashId
+-- 4) Sends Buttons.SubscribeButton(button_1, appId) to HMI during resumption
+-- 5) HMI does not response during default timeout
+-- SDL does:
+-- 1) Process empty response from HMI and sends UnsubscribeButton(button_1) to HMI
+-- 1) Respond RAI(RESUME_FAILED) to mobile app after default timeout is expaired
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/ButtonSubscription/common_buttonSubscription')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local variable ]]
+local buttonName = "PRESET_0"
+
+--[[ Local Functions ]]
+local function checkResumptionData(pAppId)
+    EXPECT_HMICALL("Buttons.SubscribeButton",
+        { appID = common.getHMIAppId(pAppId), buttonName = "CUSTOM_BUTTON" },
+        { appID = common.getHMIAppId(pAppId), buttonName = buttonName })
+    :Times(2)
+    :Do(function(_, data)
+        if data.params.buttonName ~= buttonName then
+            common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", { })
+        end
+    end)
+    EXPECT_HMICALL("Buttons.UnsubscribeButton",
+        { appID = common.getHMIAppId(pAppId), buttonName = buttonName })
+    :Do(function(_, data)
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", { })
+    end)
```

Comment:
@TMyza since we implement a test case where HMI does not respond during resumption, i think this expectation should also have an extended timeout of 12000. Please add `:Timeout(12000)` if my point is valid